mempool/mempool.go

```diff
@@ -99,27 +99,27 @@ func NewAuctionMempool(txDecoder sdk.TxDecoder, txEncoder sdk.TxEncoder, maxTx i
 				MaxTx:      maxTx,
 			},
 		),
-		txDecoder: txDecoder,
-		txEncoder: txEncoder,
-		txIndex:   make(map[string]struct{}),
-		Config:    config,
+		txDecoder:  txDecoder,
+		txEncoder:  txEncoder,
+		txIndex:    make(map[string]struct{}),
+		AuctionBid: config,
 	}
 }
 
 // Insert inserts a transaction into the mempool based on the transaction type (normal or auction).
 func (am *AuctionMempool) Insert(ctx context.Context, tx sdk.Tx) error {
-	isAuctionTx, err := am.IsAuctionTx(tx)
+	bidInfo, err := am.GetAuctionBidInfo(tx)
 	if err != nil {
 		return err
 	}
 
 	// Insert the transactions into the appropriate index.
 	switch {
```

Comment:
I think an `if/else` would be a bit cleaner here IMO. `if bidInfo != nil  {} else {}`